entry_tasks/48_type_checker/type_checker.py

```diff
@@ -0,0 +1,34 @@
+if __name__ == '__main__':
+    import re
+
+    def is_not_empty(s):
+        return bool(s and s.strip())
+
+    def reformator(listik):
+        list = []
+        for item in listik:
+            if item.isdigit():
+                list.append((int(item), type(int(item))))
```

Comment:
no sense to type(int()) since you already know that int will return a type of int :)
`type(int(item))` -> just `int`